pkg/ddl/backfilling.go

```diff
@@ -619,6 +621,146 @@ func SetBackfillTaskChanSizeForTest(n int) {
 	backfillTaskChanSize = n
 }
 
+func (dc *ddlCtx) runAddIndexInLocalIngestMode(
+	ctx context.Context,
+	sessPool *sess.Pool,
+	t table.PhysicalTable,
+	reorgInfo *reorgInfo,
+) error {
+	// TODO(tangenta): support adjust worker count dynamically.
+	startKey, endKey := reorgInfo.StartKey, reorgInfo.EndKey
```

Comment:
move them closer to usage (like line 672), or directly use `reorgInfo.StartKey` and `reorgInfo.EndKey`

Reply:
Changed to `reorgInfo.StartKey` and `reorgInfo.EndKey`. https://github.com/pingcap/tidb/pull/54149/commits/545c9e9c4bd59aa756d688ab4dae9cfdd9248d9d